lib/tbot/spiffe/trust_bundle_cache.go

```diff
@@ -557,13 +512,68 @@ func (m *TrustBundleCache) processEvent(ctx context.Context, event types.Event)
 				"x509_authorities", len(bundle.X509Authorities()),
 			)
 			bundleSet.Federated[federation.Metadata.Name] = bundle
-			m.setAndBroadcastBundleSet(bundleSet)
+			m.setBundleSet(bundleSet)
 		}
 	default:
 		log.WarnContext(ctx, "Ignoring unexpected event type")
 	}
 }
 
+func FetchInitialBundleSet(
```

Comment:
Yeah - moving it around a little bit so it can be called in a once-off nature by a caller who doesn't want to spin up the whole cache. `tbot` operates either as a daemon (in which the cache makes a lot of sense) and also in a "one-shot" mode. In "one-shot" spinning up the cache is unnecessary as the info can just be fetched once.